README.md

```diff
@@ -10,18 +10,18 @@ The `rapids_dask_dependency` package encodes both `dask` and `distributed` requi
 During the development cycle for RAPIDS, this metapackage will be released as an alpha-versioned package, e.g. `23.10.00a0`.
 
 For conda packages, RAPIDS repos will only pin up to the RAPIDS patch version, i.e. `==23.10.00.*`.
-When RAPIDS hits code freeze and we pin dask versions, the package versions in this repository should be pinned.
+When RAPIDS hits code freeze and we pin Dask versions, the package versions in this repository should be pinned.
 At this time, a non-alpha release of the metapackage will be created, `23.10.00`.
 This new metapackage version will be automatically picked up by other RAPIDS libraries since they will be using a `==23.10.00.*` pin.
 
-For pip wheels, RAPIDS repos will need to set up their wheel building scripts to add an alpha spec `>=0.0.0a0` to the dask dependencies in pyproject.toml.
+For pip wheels, RAPIDS repos will need to set up their wheel building scripts to add an alpha spec `>=23.10.00a0` to the Dask dependencies in `pyproject.toml`.
```

Comment:
This change isn't quite right. What we want to do is something more like this: https://github.com/rapidsai/ucx-py/blob/branch-0.35/ci/build_wheel.sh#L37

Reply:
To be clear, your change would work too, but then we would have the same constant churn in every RAPIDS repo where the alpha version would need to be changed each release. With this scripted approach it "just works" without any additional changes. Only the `rapids-dask-dependency` package would need to change for pinnings to take effect.

Reply:
I'm not sure I understand it though, we don't pin to a specific version then? We literally want `>=0.0.0a0`?

Reply:
Copying in my longer explanation from offline for posterity:
- For conda's dependency processing, specifying a dependency like `==23.10.00.*` will match `23.10.00` and `23.10.00a0`, but it will prefer the former (the release) over the latter (the alpha). Therefore, with conda we only need to include that spec and it will "just work"
For pip, `==23.10.00.*` will not match `23.10.00a0`. To match an alpha, you need a dependency like `>=23.10.00.a0`. However, if we put that into a package like `cudf`, then come release we have to go in and change that dependency to be `==23.10.00`, otherwise in the future someone installing `cudf==23.10.00` could pull in `rapids-dask-dependency==23.12.00` because of the `>=`. That's churn that we don't want. To prevent that, I'm proposing that we encode the dependency as `23.10.00.*`, then programatically add `>=0.0.0a0` when building nightly wheels. That will allow both `23.10.00` and `23.10.00a0` to match. For release builds we won't add that, so only `23.10.00` will match.

Reply:
Thanks Vyas for the clarifications!